src/query/Querent.js

```diff
@@ -1,17 +1,23 @@
-goog.provide('ngeo.Querent');
+/**
+ * @module ngeo query namespace
```

Comment:
what's that ? Do you know ?

Reply:
We should not put `@module` annotations; they will be generated automatically later (in a specific format).